src/operation.js

```diff
@@ -247,49 +283,38 @@ export class Operation {
         throw new Error(`${name} value is invalid`);
       case value < 0:
         throw new Error(`${name} value must be unsigned`);
-      case !isValidFunction || (isValidFunction && isValidFunction(value, name)):
+      case !isValidFunction ||
+        (isValidFunction && isValidFunction(value, name)):
         return value;
       default:
         throw new Error(`${name} value is invalid`);
     }
   }
 
-  /**
-   * @private
-   */
```

Comment:
Please leave `@private` tags. Otherwise these function will become visible in jsdocs.